src/validation/areChangeFilesDeleted.ts

```diff
@@ -1,11 +1,11 @@
-import { findGitRoot, getChangePath } from '../paths';
+import { findProjectRoot, getChangePath } from '../paths';
 import { BeachballOptions } from '../types/BeachballOptions';
 import { getChangesBetweenRefs } from '../git';
 
 export function areChangeFilesDeleted(options: BeachballOptions): boolean {
   const { branch, path: cwd } = options;
 
-  const gitRoot = findGitRoot(cwd);
+  const gitRoot = findProjectRoot(cwd);
```

Comment:
same as the other suggestion: gitRoot -> root; findProjectRoot -> getWorkspaceRoot